### PR TITLE
feat(ArraySummary): hide on negative statement

### PR DESCRIPTION
**Pull request by chohner**
Arrays where the statement was not `yes` (for example `hasKinder`) are now hidden in the array overview (instead of offering to edit the answer)
- removes statementUrl & statementValue from arrayConfig
- moves array filtering logic into `getSummaryData()` + update tests
- fixes styling (including a small bug in `<PageContent>` that spans empty divs for non-rendered entries)

Note:
- I left the default 40px horizontal gap (instead of the specified 48px)
- the labels `arrayChangeStatementButtonLabel` and `arrayStatementIsNoLabel` can be removed from strapi once this is deployed

### CODE REVIEW

File: app/services/array/getSummaryData.ts [hunk 1]
```diff
@@ -13,22 +13,16 @@ export function getSummaryData(
   return Object.fromEntries(
     categories
       .map((category) => category)
-      .filter((category) => category in arrayConfigurations)
```
Comment (frederike-ramin) [inline app/services/array/getSummaryData.ts:16]: This is now a very small function and I'm wondering if we even need it as a separate file? But no strong feelings here.

**Review by frederike-ramin — APPROVED** [review]
Just three little things and I removed the accidentally committed code. Other than that it looks good to me. :)

File: app/services/array/getSummaryData.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (chohner) [reply app/services/array/getSummaryData.ts:16]: probably legacy.. removing it

File: app/components/PageContent.tsx [hunk 2]
```diff
@@ -103,14 +104,16 @@ function PageContent({
...
-      {content.map((el) => (
-        <div key={keyFromElement(el)}>
-          {wrapInBackground(
-            el,
-            wrapInContainer(el, cmsToReact(el), fullScreen),
-          )}
-        </div>
-      ))}
+      {content
+        .filter((el) => el.__component !== "page.array-summary")
```
[the remark this answers is not in the document]
Comment (chohner) [reply app/components/PageContent.tsx:107]: I think the migration overview is not a `StrapiContentComponent` so shouldn't be an issue. The array summary however would spam empty `<div/>` if not filtered out.

File: app/services/array/getSummaryData.ts (hunk 1 above)
Comment (chohner) [reply app/services/array/getSummaryData.ts:16]: leaving it for now as it simplifies the formlar loader

**Pull request merged**